<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: #121212;
    color: #EAEAEA;
    font-family: &quot;Segoe UI&quot;;
    font-size: 14px;
}

QMainWindow::separator { background: transparent; }

QLabel#titleLabel {
    font-size: 22px;
    font-weight: 700;
    color: #FFFFFF;
}

QLabel#subtitleLabel {
    font-size: 12px;
    color: #A8A8A8;
}

QGroupBox {
    border: 1px solid #2A2A2A;
    border-radius: 12px;
    margin-top: 12px;
    padding: 14px;
    background: #171717;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 12px;
    padding: 0 6px;
    color: #CFCFCF;
    font-weight: 600;
}

QLineEdit, QSpinBox, QComboBox {
    background: #1E1E1E;
    border: 1px solid #2D2D2D;
    border-radius: 10px;
    padding: 10px 12px;
    selection-background-color: #7C3AED;
}

QLineEdit:focus, QSpinBox:focus, QComboBox:focus {
    border: 1px solid #7C3AED;
}

QLineEdit[readOnly=&quot;true&quot;] {
    background: #161616;
    color: #D6D6D6;
    border: 1px solid #2A2A2A;
}

QComboBox::drop-down {
    border: none;
    width: 34px;
}

QComboBox::down-arrow {
    image: none;
    border: 6px solid transparent;
    border-top-color: #BDBDBD;
    margin-top: 4px;
}

QComboBox QAbstractItemView {
    background: #1E1E1E;
    border: 1px solid #2D2D2D;
    selection-background-color: #7C3AED;
    outline: 0;
}

QPushButton {
    background: #1E1E1E;
    border: 1px solid #2D2D2D;
    border-radius: 12px;
    padding: 10px 14px;
    min-height: 40px;
    font-weight: 600;
}

QPushButton:hover {
    border: 1px solid #3A3A3A;
    background: #202020;
}

QPushButton:pressed {
    background: #191919;
}

QPushButton#convertButton {
    background: #7C3AED;
    border: 1px solid #7C3AED;
    color: #FFFFFF;
}

QPushButton#convertButton:hover {
    background: #6D28D9;
    border: 1px solid #6D28D9;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="inputBaseLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select the input number system:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="inputBaseSpin">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>36</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="inputNumLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter your number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputNumEdit">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="convertButton">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-top-color: rgb(149, 87, 255);
</string>
        </property>
        <property name="text">
         <string>Ð¡onvert </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="swapButton">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
          <italic>true</italic>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgb(43, 54, 255);
border-top-color: rgb(67, 86, 255);</string>
        </property>
        <property name="text">
         <string>&lt;--&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyButton">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
          <fontweight>DemiBold</fontweight>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(15, 199, 255);
border-right-color: rgb(179, 255, 182);
</string>
        </property>
        <property name="text">
         <string>Copy result</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="outputBaseLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Select the system of calculation of the result:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="outputBaseSpin">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>36</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="resultLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="resultEdit">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(224, 101, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">alternate-background-color: rgb(55, 20, 255);

font: 9pt &quot;Segoe UI&quot;;</string>
        </property>
        <property name="text">
         <string>download me!</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="catButton">
        <property name="toolTip">
         <string>Help</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/icons/max</normaloff>:/new/icons/max</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
